Zend/zend_vm_def.h

```diff
@@ -23,6 +23,8 @@
  * php zend_vm_gen.php
  */
 
+#include <stdint.h>
```

Comment:
This change is ignored (it doesn't end up in `zend_vm_execute.h`). I would recommend to add this include to `zend_vm_execute.skl` instead. (`stdint.h` is probably already included where `zend_vm_execute.h` is included, but this is still useful when looking at this file independently.)

Reply:
Oh I didn't even realize this header got added, will add to the skeleton